### PR TITLE
Member show endpoint

### LINKED ISSUE #50: As a User I can Visit a Member's Show Page - BE
As a user, I can visit the member's show page `member/:id` (or something similar),
And I can see the following information:
- [x] First and last name
- [x] Picture
- [x] Chamber
- [x] State
- [x] Title
- [x] District
- [x] Office Address
- [x] Party
- [ ] Map (extension)

- Contact Info:
  - [x] Phone Number
  - [x] Twitter Handle (link)
  - [x] YouTube Handle (link)
  - [x] FaceBook Handle (link)
  - [x] Website (link)
  - [x] Website Contact Form (link)

 
Expose the above information at the above endpoint.

### PR DESCRIPTION
**Pull request by SMJ289**
## Type of change made:
- [ ] **Bugfix** (non-breaking change which fixes an issue)
- [x] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected)
- [ ] **Refactoring** (improves existing functionality)
- [x] **Testing** (tests existing code or improves existing tests)
- [ ] **Styling** (no functionality changed, just visual improvements)
## Description:

This adds the endpoint for detailed member show page. 

## What functionality does this accomplish?
- Adds the endpoint `/member/:id/`
- Adds methods to format all social media handles as links
- Adds image to MemberIndex json response
- Adds `py-cov` for testing coverage report
**Run this to test and generate coverage report**
`pipenv run pytest --cov=app`

closes #50 

## Checklist:
- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have fully tested my code
- [x] I have partially tested my code (please explain why):
Need to add unit testing to classes.

### CODE REVIEW

**Comment by jklich151** [comment]
Created member show info for a specific member by ID. Set null for any urls that are not provided. Looks good!

File: classes.py [hunk 1]
```diff
@@ -10,27 +11,76 @@ def __init__(self,
...
+        if handle == None:
+          return None
+        else:
+          return f'https://www.facebook.com/{handle}'
+
+    def youtube_url(self, handle):
+        if handle == None:
+          return None
+        else:
+          return f'https://www.youtube.com/user/{handle}'
```
Comment (itemniner) [inline classes.py:62]: Good use of handling the nul.

**Comment by jklich151** [comment]
py cov added so we can track our testing coverage

File: classes.py [hunk 2]
```diff
@@ -10,27 +11,76 @@ def __init__(self,
                  facebook,
```
Comment (itemniner) [inline classes.py:11]: Im curious if we need to initialize everything or we can leave out some attributes like ruby/

Comment (SMJ289) [reply classes.py:11]: You have to indicate it as an optional argument or it will get mad.

**Pull request merged**